protostar/commands/init/init_command.py

```diff
@@ -42,16 +42,30 @@ def arguments(self) -> List[Command.Argument]:
                 name="existing",
                 description="Adapt current directory to a Protostar project.",
                 type="bool",
-            )
+            ),
+            Command.Argument(
+                name="name",
+                description="Name of the directory a new project will be placed in. "
+                "Ignored when `--existing` is passed.",
+                type="str",
+            ),
```

Comment:
[Functionality] I'd make it optional position argument.
`protostar init my-new-project`

Reply:
done